IMS.BL/Repositories/InventoryRepository.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections.Generic;
+using IMS.BL.Domain;
+
+namespace IMS.BL.Repositories
+{
+    public class InventoryRepository
```

Comment:
this is more of service even tho you are not using it as a service because you do not but any business logic in it